requirements.txt

```diff
@@ -1,2 +1,3 @@
 strawberryfields>=0.15
 pennylane>=0.11
+tensorflow
```

Comment:
Do we need to version pin to at least 2?
Also, the idea to not put in `setup.py` is it's a soft requirement?

Reply:
```suggestion
tensorflow>=2.0
```
>  Also, the idea to not put in setup.py is it's a soft requirement?
Yep, exactly :)